.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: clangsmith, mail2fei
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Version [e.g. 22]
```

Comment:
```suggestion
 - Version: [e.g. 22]
```

Reply:
perhaps can be more specific to us? as we don't support iOS:)
Client OS: [e.g. Windows 11]
Extension version: [e.g. v0.1.1]
SAS version: [e.g. 2022.11]

Reply:
@mail2fei, can we also include Extension version in the information. It seems like that'd be pretty helpful. I also think we need to run a `yarn format`